x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -169,24 +170,25 @@ describe('policy table', () => {
     expect(buttons.at(2).text()).toBe('Delete policy');
     expect((buttons.at(2).getDOMNode() as HTMLButtonElement).disabled).toBeTruthy();
```

Comment:
Is this testing the correct thing? It looks like the view indices button is at index `0` and disabled should be `false`.

```
expect((buttons.at(2).getDOMNode() as HTMLButtonElement).disabled).toBeFalsey();
```